esp/xslt/esdl2monitor.xslt

```diff
@@ -197,6 +197,17 @@ END;
 </xsl:if>
   requestIn := DATASET([], the_requestLayout) : STORED ('<xsl:value-of select="$requestType"/>', FEW, FORMAT(FIELDWIDTH(100),FIELDHEIGHT(30), sequence(100)));
 
+  exceptionRec := RECORD
+    string10 Source {xpath('Source')};
+    integer2 Code {xpath('Code')};
+    string100 Message {xpath('Message')};
+  END;
+
+  soapout_rec := record
```

Comment:
very minor, but the record naming convention should be consistent.
exceptionRec vs soapout_rec